deploy issues

@claude

app/api:
2025-10-19T19:37:14.242649538Z ==> Downloading cache...
2025-10-19T19:37:14.244535159Z ==> Cloning from https://github.com/muppetbrown/mana_meeples_singles_market
2025-10-19T19:37:14.798325816Z ==> Checking out commit d96067a8ae0f58a45a66de83db628d12d88bbca4 in branch main
2025-10-19T19:37:28.821085629Z ==> Transferred 80MB in 7s. Extraction took 6s.
2025-10-19T19:37:39.536300491Z ==> Requesting Node.js version 20.x
2025-10-19T19:37:39.681277468Z ==> Using Node.js version 20.19.5 via environment variable NODE_VERSION
2025-10-19T19:37:39.706904513Z ==> Docs on specifying a Node.js version: https://render.com/docs/node-version
2025-10-19T19:37:39.76865638Z ==> Running build command 'corepack enable && pnpm install --no-frozen-lockfile && pnpm run build:render'...
2025-10-19T19:37:40.386652789Z Scope: all 3 workspace projects
2025-10-19T19:37:41.130218008Z 
2025-10-19T19:37:41.130241518Z devDependencies: skipped because NODE_ENV is set to production
2025-10-19T19:37:41.130244938Z 
2025-10-19T19:37:41.144513936Z Done in 1.2s
2025-10-19T19:37:41.554062315Z 
2025-10-19T19:37:41.554088316Z > mana-meeples-singles-market@ build:render /opt/render/project/src
2025-10-19T19:37:41.554094246Z > pnpm --filter ./apps/web run build && pnpm --filter ./apps/api run build && node scripts/copy-dist-to-api.js
2025-10-19T19:37:41.554099866Z 
2025-10-19T19:37:41.97007317Z 
2025-10-19T19:37:41.970544733Z > web@0.1.0 build /opt/render/project/src/apps/web
2025-10-19T19:37:41.970548883Z > vite build
2025-10-19T19:37:41.970551913Z 
2025-10-19T19:37:42.166200595Z vite v5.4.20 building for production...
2025-10-19T19:37:42.470818196Z transforming...
2025-10-19T19:37:42.535472991Z ✓ 3 modules transformed.
2025-10-19T19:37:42.541518335Z x Build failed in 349ms
2025-10-19T19:37:42.541627588Z error during build:
2025-10-19T19:37:42.541640749Z [vite:css] [postcss] /opt/render/project/src/apps/web/src/styles/index.css:117:1: Unclosed block
2025-10-19T19:37:42.541646549Z file: /opt/render/project/src/apps/web/src/styles/index.css:117:0
2025-10-19T19:37:42.541651609Z     at Input.error (/opt/render/project/src/node_modules/.pnpm/postcss@8.5.6/node_modules/postcss/lib/input.js:135:16)
2025-10-19T19:37:42.541656169Z     at Parser.unclosedBlock (/opt/render/project/src/node_modules/.pnpm/postcss@8.5.6/node_modules/postcss/lib/parser.js:575:22)
2025-10-19T19:37:42.541660409Z     at Parser.endFile (/opt/render/project/src/node_modules/.pnpm/postcss@8.5.6/node_modules/postcss/lib/parser.js:335:35)
2025-10-19T19:37:42.541664909Z     at Parser.parse (/opt/render/project/src/node_modules/.pnpm/postcss@8.5.6/node_modules/postcss/lib/parser.js:476:10)
2025-10-19T19:37:42.541669779Z     at parse (/opt/render/project/src/node_modules/.pnpm/postcss@8.5.6/node_modules/postcss/lib/parse.js:11:12)
2025-10-19T19:37:42.54167413Z     at new LazyResult (/opt/render/project/src/node_modules/.pnpm/postcss@8.5.6/node_modules/postcss/lib/lazy-result.js:165:16)
2025-10-19T19:37:42.54167849Z     at Processor.process (/opt/render/project/src/node_modules/.pnpm/postcss@8.5.6/node_modules/postcss/lib/processor.js:53:14)
2025-10-19T19:37:42.54168347Z     at compileCSS (file:///opt/render/project/src/node_modules/.pnpm/vite@5.4.20_@types+node@20.19.22/node_modules/vite/dist/node/chunks/dep-D_zLpgQd.js:36898:59)
2025-10-19T19:37:42.54168774Z     at async Object.transform (file:///opt/render/project/src/node_modules/.pnpm/vite@5.4.20_@types+node@20.19.22/node_modules/vite/dist/node/chunks/dep-D_zLpgQd.js:36171:11)
2025-10-19T19:37:42.54171094Z     at async transform (file:///opt/render/project/src/node_modules/.pnpm/rollup@4.52.5/node_modules/rollup/dist/es/shared/node-entry.js:21139:16)
2025-10-19T19:37:42.554088797Z /opt/render/project/src/apps/web:
2025-10-19T19:37:42.554105377Z  ERR_PNPM_RECURSIVE_RUN_FIRST_FAIL  web@0.1.0 build: `vite build`
2025-10-19T19:37:42.554107677Z Exit status 1
2025-10-19T19:37:42.577807031Z  ELIFECYCLE  Command failed with exit code 1.
2025-10-19T19:37:42.608400131Z ==> Build failed 😞
2025-10-19T19:37:42.608416002Z ==> Common ways to troubleshoot your deploy: https://render.com/docs/troubleshooting-deploys

app/web:
2025-10-19T19:37:06.115214978Z ==> Downloading cache...
2025-10-19T19:37:06.117610577Z ==> Cloning from https://github.com/muppetbrown/mana_meeples_singles_market
2025-10-19T19:37:06.789497871Z ==> Checking out commit d96067a8ae0f58a45a66de83db628d12d88bbca4 in branch main
2025-10-19T19:37:21.50364512Z ==> Transferred 116MB in 8s. Extraction took 6s.
2025-10-19T19:37:25.734510531Z ==> Installing dependencies with pnpm...
2025-10-19T19:37:25.815960521Z ==> Requesting Node.js version 20.x
2025-10-19T19:37:26.645362538Z ==> Using Node.js version 20.19.5 via /opt/render/project/src/package.json
2025-10-19T19:37:26.671535615Z ==> Docs on specifying a Node.js version: https://render.com/docs/node-version
2025-10-19T19:37:27.358664588Z Scope: all 3 workspace projects
2025-10-19T19:37:28.189086291Z 
2025-10-19T19:37:28.212029735Z Done in 1.4s
2025-10-19T19:37:28.257333025Z ==> Running build command 'corepack enable && pnpm install --no-frozen-lockfile && pnpm --filter ./apps/web run build'...
2025-10-19T19:37:28.815857928Z Scope: all 3 workspace projects
2025-10-19T19:37:29.645990486Z 
2025-10-19T19:37:29.669614081Z Done in 1.1s
2025-10-19T19:37:30.183383373Z 
2025-10-19T19:37:30.183436344Z > web@0.1.0 build /opt/render/project/src/apps/web
2025-10-19T19:37:30.183439874Z > vite build
2025-10-19T19:37:30.183442584Z 
2025-10-19T19:37:30.434948501Z vite v5.4.20 building for production...
2025-10-19T19:37:30.749279215Z transforming...
2025-10-19T19:37:30.815366225Z ✓ 3 modules transformed.
2025-10-19T19:37:30.820130033Z x Build failed in 349ms
2025-10-19T19:37:30.820173853Z error during build:
2025-10-19T19:37:30.820179304Z [vite:css] [postcss] /opt/render/project/src/apps/web/src/styles/index.css:117:1: Unclosed block
2025-10-19T19:37:30.820182584Z file: /opt/render/project/src/apps/web/src/styles/index.css:117:0
2025-10-19T19:37:30.820186313Z     at Input.error (/opt/render/project/src/node_modules/.pnpm/postcss@8.5.6/node_modules/postcss/lib/input.js:135:16)
2025-10-19T19:37:30.820189194Z     at Parser.unclosedBlock (/opt/render/project/src/node_modules/.pnpm/postcss@8.5.6/node_modules/postcss/lib/parser.js:575:22)
2025-10-19T19:37:30.820192074Z     at Parser.endFile (/opt/render/project/src/node_modules/.pnpm/postcss@8.5.6/node_modules/postcss/lib/parser.js:335:35)
2025-10-19T19:37:30.820194884Z     at Parser.parse (/opt/render/project/src/node_modules/.pnpm/postcss@8.5.6/node_modules/postcss/lib/parser.js:476:10)
2025-10-19T19:37:30.820198044Z     at parse (/opt/render/project/src/node_modules/.pnpm/postcss@8.5.6/node_modules/postcss/lib/parse.js:11:12)
2025-10-19T19:37:30.820200874Z     at new LazyResult (/opt/render/project/src/node_modules/.pnpm/postcss@8.5.6/node_modules/postcss/lib/lazy-result.js:165:16)
2025-10-19T19:37:30.820203834Z     at Processor.process (/opt/render/project/src/node_modules/.pnpm/postcss@8.5.6/node_modules/postcss/lib/processor.js:53:14)
2025-10-19T19:37:30.820207534Z     at compileCSS (file:///opt/render/project/src/node_modules/.pnpm/vite@5.4.20_@types+node@20.19.22/node_modules/vite/dist/node/chunks/dep-D_zLpgQd.js:36898:59)
2025-10-19T19:37:30.820210404Z     at async Object.transform (file:///opt/render/project/src/node_modules/.pnpm/vite@5.4.20_@types+node@20.19.22/node_modules/vite/dist/node/chunks/dep-D_zLpgQd.js:36171:11)
2025-10-19T19:37:30.820226814Z     at async transform (file:///opt/render/project/src/node_modules/.pnpm/rollup@4.52.5/node_modules/rollup/dist/es/shared/node-entry.js:21139:16)
2025-10-19T19:37:30.831507958Z /opt/render/project/src/apps/web:
2025-10-19T19:37:30.831526659Z  ERR_PNPM_RECURSIVE_RUN_FIRST_FAIL  web@0.1.0 build: `vite build`
2025-10-19T19:37:30.831529809Z Exit status 1
2025-10-19T19:37:30.872242914Z ==> Build failed 😞
2025-10-19T19:37:30.872261414Z ==> Common ways to troubleshoot your deploy: https://render.com/docs/troubleshooting-deploys
2025-10-19T19:37:30.938511616Z ==> Requesting Node.js version 20.x
2025-10-19T19:37:31.103255357Z ==> Using Node.js version 20.19.5 via /opt/render/project/src/package.json
2025-10-19T19:37:31.1346774Z ==> Docs on specifying a Node.js version: https://render.com/docs/node-version

--- a/apps/web/src/styles/index.css
+++ b/apps/web/src/styles/index.css
@@ -121,15 +121,16 @@
     box-sizing: border-box;
   }
 
-/* Screen Reader Only utility */
-.sr-only {
-  position: absolute;
-  width: 1px;
-  height: 1px;
-  padding: 0;
-  margin: -1px;
-  overflow: hidden;
-  clip: rect(0, 0, 0, 0);
-  white-space: nowrap;
-  border: 0;
+  /* Screen Reader Only utility */
+  .sr-only {
+    position: absolute;
+    width: 1px;
+    height: 1px;
+    padding: 0;
+    margin: -1px;
+    overflow: hidden;
+    clip: rect(0, 0, 0, 0);
+    white-space: nowrap;
+    border: 0;
+  }
 }

